tests/jlm/llvm/backend/dot/DotWriterTests.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2024 Håvard Krogstie <krogstie.havard@gmail.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#include <test-registry.hpp>
+#include <test-util.hpp>
+#include <TestRvsdgs.hpp>
+
+#include <jlm/llvm/backend/dot/DotWriter.hpp>
+#include <jlm/llvm/ir/operators.hpp>
+
+#include <cassert>
+
+static int
+TestWriteGraphs()
+{
+  using namespace jlm::llvm;
+  using namespace jlm::util;
+
+  // Arrange
+  jlm::tests::GammaTest gammaTest;
+
+  // Act
+  GraphWriter writer;
+  dot::WriteGraphs(writer, *gammaTest.graph().root(), false);
+
+  // writer.OutputAllGraphs(std::cout, GraphOutputFormat::Dot);
```

Comment:
Let the line commented in, otherwise it will get stale. There is no problem in printing something in a test.